### ISSUE TITLE
Switch documentation and docker files to reference non-Oracle technology

### ISSUE THREAD

**Issue by cbeach47** [apereo/openEQUELLA#860]
Usage of Oracle JDK/JREs are now being impacted by licensing issues.

It's possible to build openEQUELLA including the web start and applets with open solutions such as OpenJDK / IcedTea with IcedTea-Web.  

The documentation and docker files should reflect an 'open source' solution for developing, building, and running openEQUELLA.

**Comment by edalex-ian** [comment]
Note: Our travis builds now use Open JDK.

**Comment by cbeach47** [comment]
Once PR 1046 is merged, we should cherry-pick this into the `release/2019.1` branch.

**Comment by cbeach47** [comment]
Merged into `develop` and cherry-picked into `release/2019.1`
